src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/InputPackageDiffDirectory.java

```diff
@@ -0,0 +1,8 @@
+package com.puppycrawl.tools.checkstyle.checks;
+
+public class InputPackageDiffDirectory {
```

Comment:
Would having a separate folder conflict with Issue #3933 . If not what name should be used ?. And correct me if I am wrong, by "same" you mean `package com.puppycrawl.tools.checkstyle.checks;`?